wiki/Guides/Tablet_Purchase/de.md

```diff
@@ -0,0 +1,93 @@
+
+![](img/osu!tabletv1.jpg "osu!tablet v1 \(2013\)")
+
+![](img/osu!tabletv2.jpg "osu!tablet v2 \(2016\)")
+
+### Huion
+
+Huion ist eine Tablet-Firma, die der Hersteller für das osu!tablet ist. Sie sind am besten bekannt dafür, dass sie mit großem Abstand günstigere Tablets als Wacom anbieten. Manche würden sagen, dass sich ihre Tablets schlechter anfühlen und dass sich die Verwendung eines Tablet-Stifts mit einem Huion-Tablet anfühlt, als würde man auf Papier schreiben (im Vergleich dazu sagen einige, dass es sich bei Wacom anfühlt, als würde man auf Glas schreiben). Ihre Tablet-Stifte brauchen entweder eine Batterie oder müssen geladen werden, im Gegensatz zu Tablet-Stiften von Wacom. Ebenfalls kann es sein, dass die Spitzen öfter getauscht werden müssen, als die von Wacom. Ansonsten gibt es keine wirklich sichtbaren Unterschiede zwischen Tablets von Wacom und Huion.
+
+Das HUION H430P (4096) ist vergleichbar mit dem osu!tablet (tatsächlich wird es an osu!-Spieler vermarktet).
+
+### Wacom
+
+Wacom ist der "Markenname" für Tablets. Tablets sind alles, was sie herstellen und sie sind weithin akzeptiert (von Künstlern) als eine Firma, die die beste Qualität sowie die zuverlässigsten Tablets bietet. Einige würden den Kauf bei Wacom empfehlen, da das Unternehmen als Marktführer beim Verkauf von Tablets gilt. Sie produzieren 3 Hauptlinien:
+
+- Cintiq - Tablets zum "direkt auf den Bildschirm malen". Sie werden in professionellen High-End-Anwendungen eingesetzt und kosten ein hübsches Sümmchen.
+- Intuos - Die professionelle Linie von Tablets. Diese haben Funktionen weit über die Bamboo-Linie hinaus, aber haben keinen Bildschirm.
+- Bamboo - Die generelle Linie von Tablets für Verbraucher. Diese sind günstig und haben alle Funktionen, die du brauchst, um als Hobbyist zu starten.
+
+Es gibt andere Linien von Wacom, wie z. B. die Tablets Graphire oder Volito, aber diese werden nicht weitergeführt und wurden von den oben genannten Linien ersetzt.
+
+Für osu! wirst du wahrscheinlich kein Cintiq oder Intuos benötigen, da diese beiden Linien für professionelle Arbeiten im Grafikdesign oder in der Illustration gedacht sind und vielmehr kosten als die Tablets in der Bamboo-Linie. Sie haben eine höhere Drucksensivität, mehr gebündelte Grafiksoftware und fortgeschrittene Funktionen, wie Drehung- oder Neigungssensivität. Diese kommen in osu! nicht zur Bedeutung, also kaufe dir diese nur, wenn du dein Tablet neben osu! auch für seriöses Grafikdesign verwenden möchtest.
+
+Zum Zeitpunkt der Erstellung dieses Artikels gibt es 3 Versionen von Bamboo, wobei alle verwirrend unter dem Spitznamen "Bamboo" aufgelistet werden (Die Intuos-Linien werden durch Zahlen unterschieden, Intuos5 ist der neueste aktuell). Einige würden empfehlen, entweder die zweite oder dritte Generation von Bamboo zu kaufen. Die erste hatte die zusätzlichen Buttons am oberen Rand des Tablets anstatt an der Seite, was die Benutzung in osu! erschwert.
```

Comment:
```suggestion
Wacom ist der "Markenname" für Tablets. Tablets sind alles, was sie herstellen und sie sind weithin akzeptiert (von Künstlern) als eine Firma, die die beste Qualität sowie die zuverlässigsten Tablets bietet. Einige würden den Kauf bei Wacom empfehlen, da das Unternehmen als Marktführer beim Verkauf von Tablets gilt. Sie produzieren 3 Hauptreihen:

- Cintiq - Tablets zum "direkt auf den Bildschirm malen". Sie werden in professionellen High-End-Anwendungen eingesetzt und kosten ein hübsches Sümmchen.
- Intuos - Die professionelle Reihe von Tablets. Diese haben Funktionen weit über die Bamboo-Reihe hinaus, aber haben keinen Bildschirm.
- Bamboo - Die Reihe von Tablets für allgemeine Verbraucher. Diese sind günstig und haben alle Funktionen, die du brauchst, um als Hobbyist zu starten.

Es gibt andere Reihen von Wacom, wie z. B. die Tablets Graphire oder Volito, aber diese werden nicht weitergeführt und wurden von den oben genannten Reihen ersetzt.

Für osu! wirst du wahrscheinlich kein Cintiq oder Intuos benötigen, da diese beiden Reihen für professionelle Arbeiten im Grafikdesign oder in der Illustration gedacht sind und vielmehr kosten als die Tablets in der Bamboo-Reihe. Sie haben eine höhere Drucksensivität, mehr gebündelte Grafiksoftware und fortgeschrittene Funktionen, wie Drehung- oder Neigungssensivität. Diese kommen in osu! nicht zur Bedeutung, also kaufe dir diese nur, wenn du dein Tablet neben osu! auch für seriöses Grafikdesign verwenden möchtest.

Zum Zeitpunkt der Erstellung dieses Artikels gibt es 3 Versionen von Bamboo, wobei alle verwirrend unter dem Spitznamen "Bamboo" aufgelistet werden (Die Intuos-Reihen werden durch Zahlen unterschieden, Intuos5 ist der neueste aktuell). Einige würden empfehlen, entweder die zweite oder dritte Generation von Bamboo zu kaufen. Die erste hatte die zusätzlichen Buttons am oberen Rand des Tablets anstatt an der Seite, was die Benutzung in osu! erschwert.
```
Maybe `Reihe` works better instead of `Linie`

Reply:
GitHub didn't like that there were several suggestions for one line. I will apply this one manually 😐